tests/kola/libtest.sh

```diff
@@ -0,0 +1,73 @@
+# Source library for shell script tests
```

Comment:
(At some point we should probably copy this into coreos-assembler; but definitely something after this PR)